types/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
@ericdao-ttt Could you please explain why this file and the empty file, `types/react-native-otp-inputs-tests.tsx`, is needed? Thanks.